mirest/ah-haven-space-sprinters
# #164694224 Users should comment on articles

**Description of tasks to be completed**

Have the following endpoints working

`POST api/articles/<slug>/comments`
`GET api/articles/<slug>/comments`
`GET api/articles/comment/<pk>`
`PATCH api/articles/comment/<pk>`
`DELETE api/articles/comment/<pk>`
`POST api/articles/<slug>/comments`
`GET api/articles/comment-replies/<comment_id>`
`GET api/articles/comment-replies/<comment_id>`
`PATCH api/articles/replies/<pk>`
`DELETE api/articles/replies/<pk>`



**How should this be manually tested**

- clone the repo and cd into the directory then create a virtualenv and activate 
- install all the requirements using command 'pip install -r requirements.txt'
- ensure to set all the environment variables in the .env file and the run the server using command 'python manage.py runserver'
- open postman or your favorite client and test out all the endpoints listed above
- use this data format in the body `{"comment_body":"value",}` for comment
- use this data format in the body `{"body":"value",}` for reply

Alternatively 
- You can test out the above endpoints using a link provided by the review app with the same data format

**What are the relevant pivotal tracker stories**

[#164694224](https://www.pivotaltracker.com/story/show/164694224)
